pkg/gofr/handler.go

```diff
@@ -149,8 +149,30 @@ func panicRecoveryHandler(re any, log logging.Logger, panicked chan struct{}) {
 func (h handler) logError(traceID string, err error) {
 	if err != nil {
 		errorLog := &ErrorLogEntry{TraceID: traceID, Error: err.Error()}
-		h.container.Logger.Error(errorLog)
+
+		if isHTTPError(err) {
+			h.container.Logger.Info(errorLog)
+		} else {
+			h.container.Logger.Error(errorLog)
+		}
```

Comment:
What about this?

```suggestion
		logHelper := h.container.Logger.Error
		if isHTTPError(err) {
			logHelper = h.container.Logger.Info
		}
		logHelper(errorLog)
```

Reply:
I have changed the implementation to this:

```go
func (h handler) logError(traceID string, err error) {
	if err != nil {
		errorLog := &ErrorLogEntry{TraceID: traceID, Error: err.Error()}

		if errors.As(err, &gofrHTTP.ErrorEntityAlreadyExist{}) {
			h.container.Logger.Info(errorLog)
		} else if errors.As(err, &gofrHTTP.ErrorEntityNotFound{}) {
			h.container.Logger.Info(errorLog)
		} else if errors.As(err, &gofrHTTP.ErrorInvalidParam{}) {
			h.container.Logger.Info(errorLog)
		} else if errors.As(err, &gofrHTTP.ErrorMissingParam{}) {
			h.container.Logger.Info(errorLog)
		} else {
			h.container.Logger.Error(errorLog)
		}
	}
}
```